.codegen/error_mapping.py.tmpl

```diff
@@ -0,0 +1,49 @@
+# Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+from .base import DatabricksError
+
+__all__ = ['error_mapper'{{range .ExceptionTypes}}, '{{.PascalName}}'{{end}}]
+
+{{$builtinErrors := dict
+    "DEADLINE_EXCEEDED" "TimeoutError"
+    "NOT_FOUND" "LookupError"
+    "UNAUTHENTICATED" "PermissionError"
+    "PERMISSION_DENIED" "PermissionError"
+    "TOO_MANY_REQUESTS" "ResourceWarning"
+    "NOT_IMPLEMENTED" "NotImplementedError"
+}}
```

Comment:
Ditto, let's remove this for now and let users reraise if they want to adapt our exceptions to other types.